crates/primitives/src/chain/mod.rs

```diff
@@ -116,6 +122,30 @@ impl Chain {
         Chain::Named(NamedChain::Holesky)
     }
 
+    /// Returns the optimism goerli chain.
+    #[cfg(feature = "optimism")]
+    pub const fn optimism_goerli() -> Self {
+        Chain::Named(NamedChain::OptimismGoerli)
+    }
+
+    /// Returns the optimism mainnet chain.
+    #[cfg(feature = "optimism")]
+    pub const fn optimism_mainnet() -> Self {
+        Chain::Named(NamedChain::Optimism)
+    }
+
+    /// Returns the base goerli chain.
+    #[cfg(feature = "optimism")]
+    pub const fn base_goerli() -> Self {
+        Chain::Named(NamedChain::BaseGoerli)
+    }
+
+    /// Returns the base mainnet chain.
+    #[cfg(feature = "optimism")]
+    pub const fn base_mainnet() -> Self {
+        Chain::Named(NamedChain::Base)
+    }
```

Comment:
we can get rid of the features here

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/136